scribble-lib/scribble/lp/lang/common.rkt

```diff
@@ -41,16 +41,18 @@
        (let loop ([block (get-chunk main-id)])
          (append-map
           (lambda (expr)
-            (if (identifier? expr)
-                (let ([subs (get-chunk expr)])
-                  (if (pair? subs)
-                      (begin (set! chunk-mentions (cons expr chunk-mentions))
-                             (loop subs))
-                      (list (shift expr))))
-                (let ([subs (syntax->list expr)])
-                  (if subs
-                      (list (restore expr (loop subs)))
-                      (list (shift expr))))))
+            (cond
+              [(identifier? expr)
+               (define subs (get-chunk expr))
+               (if (pair? subs)
+                   (begin (set! chunk-mentions (cons expr chunk-mentions))
```

Comment:
Might be out of scope of this PR, but this probably should be converted to `cond`.

Reply:
Yup, I'll follow up on this with a separate pass that `cond`-ifies more things. I turned off all rules except the let-to-definitions rules when generating this pull request.